File: MedirAngular/src/app/_interfaces/userui/AppointmentLookUpDto.ts
export interface AppointmentLookUpDto {
  appointmentId: string;
  prescription: string;
  medicFullName: string;
  positionName: string;
  polyclinicName: string;
  polyclinicAddress: string;
  cityName: string;
  price: number;
  time: Date;
  date: Date;
}

File: Medir.Backend/Medir.Application/Appointments/Queries/GetAppointmentList/AppointmentLookUpDto.cs
namespace Medir.Application.Appointments.Queries.GetAppointmentList
{
    public class AppointmentLookUpDto
    {
        public Guid AppointmentId { get; set; }

        public string Prescription { get; set; }

        public string MedicFullName { get; set; } = string.Empty;

        public string PositionName { get; set; } = string.Empty;

        public string PolyclinicName { get; set; } = string.Empty;

        public string PolyclinicAddress { get; set; } = string.Empty;

        public string CityName { get; set; } = string.Empty;

        public double Price { get; set; }

        public DateTime Time { get; set; }

        public DateTime Date { get; set; }
    }
}
